client/js/components/month/month-eventcontainer.js

```diff
@@ -1,18 +1,18 @@
 /** @jsx React.DOM */
 var React = require('react');
-var Event = require('./month-event.js');
+var MonthWeek = require('./month-week-board.js');
+var _  = require('underscore');
 
 var MonthEventContainer = React.createClass({
   render:function() {
+    var monthWeeks = _.map(this.props.monthEvents, function(week, key) {
+      return <MonthWeek key={ key } week={ week } />;
```

Comment:
So each one of these is actually going to contain a weeks worth of events. We don't get down into days till we get to the next component (currently called week-board but named to be changed in PR revision)